### PR TITLE
Nitro Botany Removal

### PR DESCRIPTION
**Pull request by MrLostman**
## About The Pull Request
Forgive me lord for I have sinned

## Changelog
:cl:
balance: removed fluorosulphuric acid from death nettle
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
I like the idea, but maybe do it without removing acid from nettle. Make some other ingredient harder to get

**Comment by MrLostman** [comment]
That was my first idea but botany is so fast these days with them having a botany chem unit moving the chemicals around only slows them down a tiny bit.

Nitro is also a very simple explosive only needing 3 chems so you cant even really spread it that far.

**Comment by MrLostman** [comment]
I'm sure that with having sep chems there are some autistic mixes people can come up with. The problem is nitro is so easy and so dam good it why would anyone experiment with anything else?

**Comment by TheFakeElon** [comment]
Just make it so nitro can’t explode in the body like most other explosive chems.

**Comment by MrLostman** [comment]
exploding in the body is not the problem is the grenades

**Comment by yorii** [comment]
Just add some random chem to the recipe instead? This is a stupid way to do this.

**Comment by MrLostman** [comment]
That would nerf nitro as a chem. Really this is only a problem because of my buff we could go back to useless sep chems @yorii

**Comment by yorii** [comment]
> That would nerf nitro as a chem. Really this is only a problem because of my buff we could go back to useless sep chems @yorii

Just add some simple chem which is not normally available from botany traits, like Nitrogen.

**Comment by yorii** [comment]
Or another way of doing it would be to rearrange the percentages of the traits, like making corn oil 50% instead of 20%, which would make it a hard requirement that you can only use omega weed to grow grenades.

**Comment by MrLostman** [comment]
Nerfing things into to the dirt is not a practice I really subscribe to. Would rather remove something than make it useless.

It's a one hit weapon and the best breaching tool that comes out in 20mins. If you leave nitro botany it is only a matter of time before people get fed up and remove sepchems.

Nitro botany can be fixed by either reverting to useless sep chems losing all the depth it offeres to botany or I can remove nitro botany and keep full power sep chems.

Adding new features to botany is out of the scope of this PR but I'm not adversed to coming up with a new system to add nitro back in, but if it was to be around as strong as it currently is it would need to take 50-70mins to get.

**Comment by TheFakeElon** [comment]
> Nerfing things into to the dirt is not a practice I really subscribe to

I don’t have a problem with nitro nerf. But removing fluro acid from death nettle is nerfing it into the ground. Won’t be worth making unless you have a lot of time and strange seeds.

Tldr; Nitro nerf = good. nettle nerf = bad

**Comment by TheFakeElon** [comment]
Just make it so nitroglycerin needs a non-room temperature to be produced.

**Comment by MrLostman** [comment]
@TheFakeElon what? removign fluro from nettle stops nothing about them? The still inject chems they still do 40 damage

**Comment by TheFakeElon** [comment]
Flurosulf is what made them gamer

**Comment by TheFakeElon** [comment]
flurosulf had uses in gas grenades too, it’s not just used for nitroglycerin

**Comment by MrLostman** [comment]
@TheFakeElon 
image
image
is it?

**Comment by TheFakeElon** [comment]
Yeah I am silly. It can be used in gas grenades to burn people and cover them in acid, if you don’t out flurosulf and clf3 in your gas melons you can’t call yourself a botanist

**Comment by MrLostman** [comment]
uuuurrreeeeeeeee right I'll close this

**Comment by TheFakeElon** [comment]
Smh, just add a temp requirement on nitro

**Comment by TheFakeElon** [comment]
Literally one line. Or like 10 if you want to make it require cold temps for muh realism

**Comment by MrLostman** [comment]
oh no I plan to do something about this, just make a different pr and talk about it more on the discord

**Comment by TheFakeElon** [comment]
Oh alright!
